Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-alpine3.18
+LABEL maintainer="roman"
+
+ENV PYTHOUNNBUFFERED=1

```

Comment:
There is a typo in the environment variable `PYTHOUNNBUFFERED`. It should be `PYTHONUNBUFFERED` to ensure that Python output is sent straight to the terminal without being buffered.